api-schema/src/main/java/org/stellar/anchor/api/platform/HealthCheckResult.java

```diff
@@ -0,0 +1,5 @@
+package org.stellar.anchor.api.platform;
+
+public interface HealthCheckResult {
+  String name();
+}
```

Comment:
What about adding a `List<Any> getStatuses()`? I think it would be more representative of how it'll be used.

Reply:
I think we can use `List<String>` to reduce the complexity for now like the way ElasticSearch does [here.](https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-health.html#cluster-health-api-response-body) 

I had added two functions in the interface: `List<String> getStatuses` and `String getStatus()` with some comments in the interface.